www/src/components/code-block/__tests__/index.js

```diff
@@ -16,7 +23,7 @@ describe(`basic functionality`, () => {
       expect(queryByText(`copy`)).toBeDefined()
     })
 
-    it(`copies text to clipboard`, () => {
+    it.only(`copies text to clipboard`, () => {
```

Comment:
Should we revert only?
```suggestion
    it(`copies text to clipboard`, () => {
```

Reply:
Yasss -- awesome catch, thank you!